hail/python/hail/utils/struct.py

```diff
@@ -114,6 +114,30 @@ def annotate(self, **kwargs):
         -------
         :class:`.Struct`
             Struct with new or updated fields.
+
+
+        Examples
+        --------
+        
+        Define a Struct `s`
+        
+        >>> s = hl.Struct(**{'food': 8, 'fruit': 5})
```

Comment:
OK, this is all really great but I think using the `**` syntax is a little confusing for a lot of our users. It's pretty advanced stuff! Can we change to the equivalent call:
```
hl.Struct(food=8, fruit=5)
```
The other examples need to change in the same way.

Reply:
I think it's not a bad idea to show an example of the `**` way, if not the first example.

Reply:
Fixed